api/src/controllers/extensions.ts

```diff
@@ -47,7 +46,7 @@ router.get(
 		res.setHeader('Content-Type', 'application/javascript; charset=UTF-8');
 		res.setHeader(
 			'Cache-Control',
-			env.EXTENSIONS_CACHE_TTL ? getCacheControlHeader(req, ms(env.EXTENSIONS_CACHE_TTL as string)) : 'no-store'
+			env.EXTENSIONS_CACHE_TTL ? `max-age=${ms(env.EXTENSIONS_CACHE_TTL as string) / 1000}` : 'no-store'
```

Comment:
We should add `EXTENSIONS_CACHE_TTL` to the `typeMap` as `string`, preventing a `NaN` value for `max-age`.

https://github.com/directus/directus/blob/de41cc5148522b2714b60061f6f1af7d2d499ea7/api/src/env.ts#L277-L281

Here's a relevant conversation: https://github.com/directus/directus/pull/17309/files#r1090772083

Reply:
Exactly! I'm on it 👍

Reply:
I'll address this in a separate pull request since other places are affected as well 👍